.github/workflows/test.yml

```diff
@@ -1,13 +1,18 @@
 name: Node CI
```

Comment:
I removed all the mentions about tokens here.

Reply:
`secrets.GRAFANA_ACCESS_POLICY_TOKEN`, and `secrets.GRAFANA_API_KEY` appear to still exist in this file. Is that an issue?

Reply:
I removed all the code that used those tokens. Can you point me where they are mentioned?